test/KeyCode.spec.js

```diff
@@ -48,4 +48,463 @@ describe('KeyCode', () => {
         assert(keyPressed === KeyCode[randomItem.name]);
     });
 
+    it('KEY_CANCEL should return 3', () => {
```

Comment:
@frameshft Can we change the wording `should return` to `equals` and use it for all these tests? Something like this `KEY_CANCEL equals 3`?